src/Commands/Site/CreateCommand.php

```diff
@@ -72,7 +72,20 @@ public function create($site_name, $label, $upstream_id, $options = ['org' => nu
         if ($site = $this->getSiteById($workflow->get('waiting_for_task')->site_id)) {
             $this->log()->notice('Deploying CMS...');
             $this->processWorkflow($site->deployProduct($upstream->id));
-            $this->log()->notice('Deployed CMS');
+            $this->log()->notice('Waiting for site availability...');
+            $waits = 0;
+            do {
+                sleep(5);
+                $woke = $site->getEnvironments()->get('dev')->wake();
+                // if success is empty, then the site is still waking up.
+                $waits++;
+                // If we've waited more than 25 seconds, then something is wrong.
+            } while (($woke['success'] ?? false) !== true || $waits < 5);
+            if (($woke['success'] ?? false) !== true) {
+                $this->log()->error('There was a problem creating the site.');
```

Comment:
We should wordsmith the error message a bit. The site might actually be fine; maybe say something along the lines of "Could not confirm that the site is working; there might be a problem." @scottbuscemi do you have an opinion?